lib/harvester.js

```diff
@@ -529,7 +529,9 @@ Harvester.prototype.noIndex = function (name) {
  *
  * harvester.setAuthorizationStrategy(function(req){
  *     //Here's all possible return types and how they should be handled.
- *     return promise;//resolved means authorized, rejected means not authorized. Return 403 jsonapi error.
+ *     return promise;//resolved means authorized,
```

Comment:
resolved no longer means authorized - it only means authorized if you don't return an error or jsonapi error.